### DISCUSSION TITLE
Soapy device query string

### DISCUSSION THREAD

**Discussion by ImDroided** [cariboulabs/cariboulite#103] [Q&A]
Can someone please help me with a proper soapy device string for apps like rtl_433. Ive looked around and have not found anything of help.

**Comment by benlamonica** [comment]
I think -d "" should work, except that there is a bug with the compiler used to compile rtl_433 that they are running into (see: https://github.com/merbanan/rtl_433/issues/2046)

I got around it by running SoapyRemote.

```
sudo SoapySDRServer --bind=127.0.0.1:1234
```

And 
```
rtl_433 -d "driver=remote,remote=tcp://127.0.0.1:1234,channel=HiF,device_id=1"
```
The channel=HiF,device_id=1 will have it pick the 6ghz antenna.

However, I don't know how well it's working, as I get nonsensical values...
```
time      : 2023-04-09 16:49:59
model     : Sharp-SPC775 id        : 184
battery_ok: 0            Temperature: 273.3 C      Humidity  : 227 %         Integrity : CRC
```
